src/Gelf/Message.php

```diff
@@ -297,6 +297,15 @@ public function toArray()
         }
 
         // return after filtering false, null and empty strings
-        return array_filter($message, 'strlen');
+        return array_filter($message, array($this, 'filter'));
+    }
+
+    /**
```

Comment:
I'm not sure if filtering is needed at all. I wouldn't strip any empty fields if I was the author.